app/marian-decoder-new.cpp

```diff
@@ -0,0 +1,61 @@
+#include <cstdlib>
```

Comment:
Now that you have this file as a main application for Service class, why do we need `app/main-mts.cpp` file? That file can be deleted. Right?

Reply:
I'd rather keep it, it doesn't hurt anything.  Also:

> Integration failures are detected if same arguments work with service-cli and does not with bergamot-translator-app.

(doc/marian-integration.md)

Reply:
The decoder is a different frontend to marian-decoder. This serves the purpose of comparing marian-decoder (upstream) with Service, for benchmarking. This is not the main application for Service.

Reply:
@jerinphilip Currently, there are 3 different main files in `app` folder of this repository. Technically, the app folder should contain only one that should demonstrate the use of unified APIs (which is done in `main.cpp`)

=> `main-mts.cpp` and this file will have to go away at sometime in future.